### DISCUSSION TITLE
I don't understand disapproved requested chains

### DISCUSSION THREAD

**Discussion by matthew2099** [WalletConnect/WalletConnectUnity#219] [Q&A]
I use WalletConnect.Instance.ConnectAsync method.
So i think connected chain.
This error message i don't understand.

![image](https://github.com/WalletConnect/WalletConnectUnity/assets/109068040/1ffe96e0-3cb0-47de-82f2-325ccb919465)

**Comment by skibitsky** [comment]
Do you use optional or required namespaces?

When using the required namespaces, the wallet has to approve all chains and methods. If the wallet doesn't support a certain method or chain, it will disapprove the connection. With optional namespaces, only supported methods and chains will be approved.
